docs/examples/Electromagnetic Phenomena/emwave_animation.py

```diff
@@ -0,0 +1,169 @@
+"""
+===============================================
+Electromagnetic Wave Propagation Animation
+===============================================
+
+A linearly polarized sinusoidal electromagnetic wave, propagating in the
+direction +x through a homogeneous, isotropic, dissipationless medium,
+such as vacuum. The electric field (blue arrows) oscillates in the
+±z-direction, and the orthogonal magnetic field (red arrows) oscillates in
+phase with the electric field, but in the ±y-direction.
+
+Function of the sinusoid used in the animation = sin(k*x - w*t + d)
+Where, k:wavenumber, x:abscissa, w:angular frequency, t:time, d:phase angle
+
+"""
+
+###############################################################################
+# Importing necessary modules
+from fury import window, actor, utils, ui
+import numpy as np
+import itertools
+
+###############################################################################
+# function that updates and returns the coordinates of the waves which are
+# changing with time
+
+
+def update_coordinates(k, w, t, d):
```

Comment:
In general, you should avoid to have 1 letter as a variable name. Even if it is really nice that you describe them above. Can you update the whole file with the correct name. Thank you!

Reply:
I renamed the variables. Thank you!